applogic/src/logging/mod.rs

```diff
@@ -60,18 +60,18 @@ fn do_init_logger(
 
     #[cfg(any(target_os = "android", target_os = "ios"))]
     {
-        registry
+        let _ = dbg!(registry
             .with(dart::layer())
             .with(fmt::Layer::new().with_writer(log_file))
-            .try_init()?;
+            .try_init());
     }
 
     #[cfg(any(target_os = "linux", target_os = "macos", target_os = "windows"))]
     {
         use fmt::writer::MakeWriterExt;
-        registry
+        let _ = dbg!(registry
             .with(fmt::Layer::new().map_writer(|w| w.and(log_file)))
-            .try_init()?;
+            .try_init());
```

Comment:
I had issues with the logger. This change allows the application to start even if the logger fails.

Reply:
Might be worth figuring out what the issue was. What kind of error did you get?